ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -132,6 +132,7 @@ def test_mean(*, dtype_and_x, keep_dims, test_flags, backend_fw, fn_name, on_dev
         on_device=on_device,
         rtol_=1e-1,
         atol_=1e-1,
+        tolerance_dict={"bfloat16": 1e-1},
```

Comment:
There's no point in adding this specific tolerance since already the rtol is 1e-1.